static/app/gettingStartedDocs/node/awslambda.tsx

```diff
@@ -12,9 +13,33 @@ import {
 } from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
 import {getJSServerMetricsOnboarding} from 'sentry/components/onboarding/gettingStartedDoc/utils/metricsOnboarding';
 import {t, tct} from 'sentry/locale';
+import {trackAnalytics} from 'sentry/utils/analytics';
 import {getInstallConfig, getSdkInitSnippet} from 'sentry/utils/gettingStartedDocs/node';
 
-type Params = DocsParams;
+export enum InstallationMode {
```

Comment:
I find this repetition very verbose and have the feeling we could make it more reusable 🤔

Reply:
Personally I would define an enum that is tied to this integration setup use-case and import it here and the two docs files + use it to check the param value in `setupDocs`